lang/en.json

```
@@ -750,6 +750,10 @@
     "Label": "Enchant Prompt",
     "Hint": "Disable the enchantment prompt when item is used. First available enchantment will always be applied."
   },
+  "Riders": {
+    "Label": "Ride Along Effects",
+    "Hint": "These additional effects will be added when this enchantment is added, and removed when it is removed."
```

Not a huge fan of the 'ride along' term. Maybe call these 'Additional Effects', or even 'Creature Effects' to be very clear, but I am open to other suggestions.

```suggestion
    "Label": "Creature Effects",
    "Hint": "These additional effects will be added to the creature when one of its items is enchanted, and removed when the enchantment is removed."
```

---

I'll go with "Additional Effects", because these are applied to the enchanted item and not necessarily transferred to the item's owner.